include/revng/CliftTransforms/BooleanNegations.td

```diff
@@ -0,0 +1,51 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
```

Comment:
In general all the transforms in this file feel like they should be part of canonicalization.